### PR TITLE
i18(es: Updated `unsupported-image-format.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `unsupported-image-format.mdx` based on this commit f61156fd82df2b15ef60140490c821ee051e1811

### CODE REVIEW

File: src/content/docs/es/reference/errors/unsupported-image-format.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
-> **UnsupportedImageFormat**: Recibí un formato no admitido `FORMAT` de `IMAGE_PATH`. Actualmente solo se admiten SUPPORTED_FORMATS.JOIN(', ') para optimización.
+> **UnsupportedImageFormat**: Recibí un formato no admitido `FORMAT` de `IMAGE_PATH`. Actualmente solo se admiten SUPPORTED_FORMATS.JOIN(', ') por nuestros servicios de imagen.
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/unsupported-image-format.mdx:7]:
```suggestion
> **UnsupportedImageFormat**: Se recibió un formato no compatible `FORMAT` desde `IMAGE_PATH`. Actualmente solo se admiten SUPPORTED_FORMATS.JOIN(', ') por nuestros servicios de imagen.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**